src/components/File.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import ExportContainer from './ExportContainer';
+import LanguageContainer from './LanguageContainer';
+import ExamplesContainer from './ExamplesContainer';
+
+import './File.scss';
+
+export default function File() {
+  const selectedLanguage = useSelector((state) => state.selectedLanguage);
+
+  return (
+    <div className="file__container">
+      <div className="languages">
```

Comment:
another convention with BEM would be to make this class something like `file__languages`

Reply:
I think the same should be done for `operations` and `actions`